Modules/binascii.c

```diff
@@ -523,42 +545,40 @@ binascii.b2a_base64
     data: Py_buffer
     /
     *
+    wrapcol: size_t = 0
```

Comment:
I suggest we restrict the value to uint32_t - allowing huge size_t values makes no sense so we should not support that.

Reply:
Except for the no-wrap case, there shouldn't be a practical possibility but we need to support `sys.maxsize` to be equivalent to "nowrap". Or maybe `None` as well. I don't know whether we have a special casing for `textwrap` when using `sys.maxsize` for instance).

Reply:
keeping it to support sys.maxsize makes sense.